modules/network-modify-config.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+// * networking/configuring-network-operator.adoc
+
+[id="network-modifying-config-{context}"]
+= Modifying network configuration in a running cluster
+
+The Network Operator reconciles the state of the cluster against a desired
+configuration.
+
+You can modify only `kubeProxy` configuration parameters in a running cluster.
+
+.Prerequisites
+
+* `oc` command installed.
+* Logged in to a running cluster.
+
+.Procedure
+
+. Use the following command to edit the `Network.operator.openshift.io` Custom
+Resource (CR):
++
+----
+$ oc edit network
+----
++
+The following example output shows a `Network.operator.openshift.io` CR that has
+been configured with default values.
++
+[source,yaml]
+----
+# Please edit the object below. Lines beginning with a '#' will be ignored,
+# and an empty file will abort the edit. If an error occurs while saving this file will be
+# reopened with the relevant failures.
+#
+apiVersion: config.openshift.io/v1
+kind: Network
+metadata:
+  creationTimestamp: 2019-04-02T17:19:32Z
+  generation: 1
+  name: cluster
+  resourceVersion: "69574"
+  selfLink: /apis/config.openshift.io/v1/networks/cluster
+  uid: 7b17364e-556b-11e9-ae15-0ab9869092cc
+spec:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+status:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  clusterNetworkMTU: 8951
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+----
+
+. Add the following code to the `spec` field in the
+`NetworkConfig.networkoperator.openshift.io` CR:
++
+[source,yaml]
+----
+spec:
+  deployKubeProxy: false
+  kubeProxyConfig:
+   iptablesSyncPeriod: 30s
+   bindAddress: 0.0.0.0
+   proxyArguments:
+   - "iptables-min-sync-period": ["30s"]
+----
+
+. Save the file and quit the text editor.
+
+The Network Operator checks the syntax when you quit the file. If your
+modifications contain a syntax error, the editor will re-open the file and
+display an error message at the top.
```

Comment:
It would be good to add a bit about checking the operator status to see if your changes were accepted.

Reply:
@squeed, how do I check the operator status?

Reply:
`oc get clusteroperator network`